src/test/clear-button.test.js

```diff
@@ -0,0 +1,34 @@
+// jest.mock('../clearTodos')
+
+import Todo from '../TodoClass.js';
+// import { clearTodos } from '../todoCrud.js';
+import createTodoItem from '../CreateTodoItem.js';
+
+describe('Test clear button ', () => {
+  test('Make test for clear button', () => {
+    document.body.innerHTML = `  
+        <ul class id="todo-container">
+
+        </ul>
+        <button id="clear-button"> 
+        </button>
+       `;
+    const todo = new Todo('Make tests for this!!');
+    const todoNode = createTodoItem(todo);
+    const todoContainer = document.getElementById('todo-container');
+    todoContainer.appendChild(todoNode);
+    const todoCheckbox = document.querySelector('.todo-checkbox');
+    todoCheckbox.click();
+    const todoButton = document.getElementById('clear-button');
+
+    const clearTodos = jest.fn(() => {
+      Todo.clearCompleted();
+      todoContainer.innerHTML = '';
+    });
+
+    todoButton.addEventListener('click', clearTodos);
+    // console.log('completed', todo.isCompleted);
```

Comment:
- [ ] Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile: